apps/ledger-live-mobile/src/components/SelectFeesStrategy.tsx

```diff
@@ -73,6 +76,12 @@ export default function SelectFeesStrategy({
 
   const onPressStrategySelect = useCallback(
     (item: FeeStrategy) => {
+      track("button_clicked", {
+        button: item.label,
+        flow: "swap",
+        page: "Swap quotes",
+        swapVersion: SWAP_VERSION,
```

Comment:
Can we create a default swap property object an export it across any track? (If tomorrow, we want to change the property the change will be done only in one place)
swapVersion, flow

Reply:
Can do, I thought about this but wasn't sure if it was worthwhile as there will only be two fields in that shared object.

Reply:
https://github.com/LedgerHQ/ledger-live/pull/2255/commits/6bf6cb751e5882133d7cc8a09edf24853bbf8fc9